R/jobs.R

```diff
@@ -604,7 +609,7 @@ job_download <- function(job,
   gs_copy(source, destination, TRUE, echo = TRUE)
 
   # write cloudml properties to run_dir
-  run_dir <- file.path(destination, basename(source))
+  run_dir <- destination
```

Comment:
I think this causes breakage for tfruns (the run_dir needs to be the fully qualified directory not just the parent "runs"

Reply:
Looks like `file.path(destination, basename(source))` used to use the source as in the GS bucket, say `gs://bucket/runs/cloudml-123123`; however, to predictably download this to the right local directory with trials we use `gs://bucket/cloudml-123123/runs/*` instead to which the `basename()` would be `*`.

I added a comment to `job_status_trial_dir()` which hopefully makes this clearer, basically, that function is our helper function to choose the right source and destination, source is either `gs://bucket/cloudml-123/runs/` or `gs://bucket/runs/cloudml-123/runs/1/*` and destination is either `runs/cloudml-123` or `runs/cloudml-123-001` when trials are available.